go/saltpack/key.go

```diff
@@ -60,9 +65,28 @@ type BoxSecretKey interface {
 	Precompute(sender BoxPublicKey) BoxPrecomputedSharedKey
 }
 
-// Keyring is an interface used with decryption; it is call to recover
-// public or private keys during the decryption process. Calls can block
-// on network action.
+// SigningSecretKey is a secret NaCl key that can sign messages.
+type SigningSecretKey interface {
+	// Sign signs message with this secret key.
+	Sign(message []byte) ([]byte, error)
```

Comment:
Should we specify that this returns just the 64-byte signature, rather than [the default NaCl thing](http://nacl.cr.yp.to/sign.html) that concatenates the signed plaintext?